extensions/amp-ad/0.1/test/test-amp-ad.js

```diff
@@ -136,8 +136,8 @@ describes.realWin('Ad loader', {amp: true}, env => {
         meta.setAttribute('name', 'amp-3p-iframe-src');
         meta.setAttribute('content', 'https://example.com/remote.html');
         doc.head.appendChild(meta);
-        a4aRegistry['zort'] = () => {
-          throw new Error('predicate should not execute if remote.html!');
+        a4aRegistry['zort'] = (win, element, useRemoteHtml) => {
```

Comment:
Nit: This doesn't need braces or an explicit `return`.

Reply:
done